Calculator/Calculator_base.cpp

```diff
@@ -218,14 +333,18 @@ int CCalculator::parseQuestion()
 		break;
 		case '(':
 		{
-			// TODO : 괄호가 시작되었을 경우에 대한 처리
-		}
-		break;
-		case ')':
-		{
-			// TODO : 괄호가 닫혔을 때에 대한 처리
+			// TODO : 괄호가 시작되었을 경우에 대한 처리 (가 나올 경우 : 클래스를 재귀호출해 답을 읽어와 숫자로 저장
+			calcute2();
```

Comment:
지금 호출한 건 `함수`지, `클래스`가 아닙니다.
그리고 유사한 코드로 함수를 새로 만들어 호출하는 건 재귀호출이라고 보기 어렵습니다.
아래와 같은 형태로 만들어 보도록 합시다.
```suggestion
		Calculator subCalculator; // 클래스를 재귀호출
		subCalculator.SetQuestion(/*괄호 안쪽의 내용*/);
		subCalculator.Calculate(); // 재귀호출한 클래스에게 괄호 안쪽의 내용을 연산시킴
		numberInput = std::to_string(subCalculator.GetAnswer()); // 결과를 받아와 numberInput에 저장
```